contracts/protocol/PositionManager.sol

```diff
@@ -288,28 +288,26 @@ contract PositionManager is
             );
             hasLiquidity = liquidityBitmap.hasLiquidity(_pip);
         }
+        uint128 remainingSize = _size - uint128(sizeOut);
         if (_size > sizeOut) {
             if (
                 _pip == _singleSlot.pip &&
                 _singleSlot.isFullBuy != (_isBuy ? 1 : 2)
             ) {
                 singleSlot.isFullBuy = _isBuy ? 1 : 2;
             }
-            //TODO validate pip
-            // convert tick to price
             // save at that pip has how many liquidity
             orderId = tickPosition[_pip].insertLimitOrder(
-                _size - uint128(sizeOut),
+                remainingSize,
                 hasLiquidity,
                 _isBuy
             );
             if (!hasLiquidity) {
-                //set the bit to mark it has liquidity
+                // set the bit to mark it has liquidity
                 liquidityBitmap.toggleSingleBit(_pip, true);
             }
         }
-        // TODO update emit event
-        emit LimitOrderCreated(orderId, _pip, _size, _isBuy);
+        emit LimitOrderCreated(orderId, _pip, remainingSize, _isBuy);
```

Comment:
should emit only the size which was added into manager's orderbook instead of the full order size from user's request (might included the market filled part). we have emitted full order size from position house